router/core/operation_planner.go

```diff
@@ -66,7 +66,10 @@ func (p *OperationPlanner) preparePlan(requestOperationName []byte, requestOpera
 		return planWithMetaData{}, &reportError{report: &report}
 	}
 
-	planner := plan.NewPlanner(context.Background(), p.executor.PlanConfig)
+	planner, err := plan.NewPlanner(p.executor.PlanConfig)
+	if err != nil {
+		return planWithMetaData{}, err
```

Comment:
Should we return pointer to planWithMetaData, so we can return nil?

Reply:
updated